scanner/src/main/kotlin/experimental/ScanController.kt

```diff
@@ -188,7 +193,7 @@ class ScanController(
     /**
      * Return true if [ScanResult]s for the complete [NestedProvenance] of the package are available.
      */
-    fun hasCompleteScanResult(scanner: PackageScannerWrapper, pkg: Package): Boolean =
+    fun hasCompleteScanResult(scanner: ScannerWrapper, pkg: Package): Boolean =
```

Comment:
Just double-checking while skimming through the code: Is this change intended?

Reply:
Yes, but I have moved it to the commit that introduced the function because `ScannerWrapper` made more sense from the beginning.